torchvision/transforms/v2/functional/_color.py

```diff
@@ -557,6 +557,25 @@ def autocontrast(inpt: datapoints._InputTypeJIT) -> datapoints._InputTypeJIT:
         )
 
 
+def equalize(inpt: datapoints._InputTypeJIT) -> datapoints._InputTypeJIT:
+    if not torch.jit.is_scripting():
+        _log_api_usage_once(equalize)
+
+    if torch.jit.is_scripting() or is_simple_tensor(inpt):
+        return equalize_image_tensor(inpt)
+    elif isinstance(inpt, PIL.Image.Image):
+        return equalize_image_pil(inpt)
```

Comment:
I would love that, since we would remove the special casing. I don't think JIT will an issue as long as we have a 

```python
if torch.jit.is_scripting():
    return ...
```

at the top.